### PR TITLE
Preferred content language getter

### PR DESCRIPTION
**Pull request by Ninerian**
Adds the getBranchDefaultLanguage and the getPreferedContentLocale functions to the API.

### CODE REVIEW

File: src/lib/app.js [hunk 1]
```diff
@@ -75,9 +75,7 @@ export async function openLinkInternal(url) {
  * Open a native file upload dialog on device which do not support it by default.
  *
  * Works only for android now
- *
- * @param {string} url the url to open in the browser
- *
+ * *
```
Comment (staust) [inline src/lib/app.js]: - [ ] fix doc

File: src/lib/connection/connector/postmessage.js [hunk 2]
```diff
@@ -33,7 +33,8 @@ const dataStore = ({ platform, language }) => ({
   native: platform.native,
   ios: platform.native === 'ios',
   android: platform.native === 'android',
-  langInfos: language
+  langInfos: language,
+  branchDefaultLang: language.branchDefaultLanguage
```
Comment (staust) [inline src/lib/connection/connector/postmessage.js]: why isn't this written out? "branchDefaultLang"

File: src/lib/app.js (hunk 1 above)
Comment (Ninerian) [reply src/lib/app.js]: What do you want to change? @staust

Comment (staust) [reply src/lib/app.js]: just remove the second start

**Pull request merged**